src/blueapi/cli/cli.py

```diff
@@ -143,10 +143,11 @@ def on_event(
         converted = json.dumps(event.dict(), indent=2)
         print(converted)
 
-    print(
-        "Subscribing to all bluesky events from "
-        f"{config.stomp.host}:{config.stomp.port}"
-    )
+    if config.stomp is not None:
+        print(
+            "Subscribing to all bluesky events from "
+            f"{config.stomp.host}:{config.stomp.port}"
+        )
```

Comment:
Must: This command does nothing without a message bus so should probably just raise if there's no stomp config